docs/book/aya/aya-tool.md

```diff
@@ -15,9 +15,10 @@ we also need that definition in Rust, not in C.
 That's what aya-tool is designed for. It's a tool which allows to generate Rust
 bindings for specific kernel structures.
 
-It can be installed with the following command:
+It can be installed with the following commands:
 
 ```console
+$ cargo install bindgen-cli
```

Comment:
We should probably add it as a dependency of aya-tool - we shouldn't require people to `cargo install` stuff manually.

Reply:
https://github.com/aya-rs/aya/issues/518

Reply:
per @alessandrod in https://github.com/aya-rs/aya/issues/518 , `bindgen-cli` is required.